### PR TITLE
Daniel steps

### PR DESCRIPTION
**Pull request by danielhuang74**

### CODE REVIEW

**Comment by YuanZhu-apple** [comment]
Thanks for interesting in contributing!
Can you explain this PR's aim and plan?

File: Asthma/Startup/APHAppDelegate.m [hunk 1]
```diff
@@ -124,12 +144,12 @@ - (void)performMigrationAfterDataSubstrateFrom:(NSInteger) __unused previousVers
 - (void) setUpInitializationOptions
 {
     
-    [APCUtilities setRealApplicationName:@"Asthma Health"];
+    [APCUtilities setRealApplicationName:@"Asthma Health+"];
```
Comment (ghost) [inline Asthma/Startup/APHAppDelegate.m:147]: Name change should only be in fork.

File: Asthma/Startup/APHAppDelegate.m [hunk 2]
```diff
@@ -97,6 +98,11 @@ - (void)performMigrationAfterDataSubstrateFrom:(NSInteger) __unused previousVers
         {
             APCLogEvent(@"Migration from version %@ to %@ has failed.", [defaults objectForKey:previousVersionKey], @(kTheEntireDataModelOfTheApp));
         }
+        
+        if (![self performMigrationFromThreeToFourWithError:&migrationError])
```
Comment (ghost) [inline Asthma/Startup/APHAppDelegate.m:102]: Migration code will be removed on PR#3 as new scheduler in AppCore makes it obsolete.

File: Asthma/Startup/APHAppDelegate.m [hunk 3]
```diff
@@ -180,12 +201,17 @@ -(void)setUpTasksReminder{
...
+    APCTaskReminder *dailyFVCSurveyReminder = [[APCTaskReminder alloc]initWithTaskID:kDailyFVCTaskId reminderBody:NSLocalizedString(@"Daily Lung Function Test", nil)];
+    
+    
     //define completion as defined in resultsSummary
     NSPredicate *medicationPredicate = [NSPredicate predicateWithFormat:@"SELF.integerValue == 1"];
     APCTaskReminder *medicationReminder = [[APCTaskReminder alloc]initWithTaskID:kDailySurveyTaskID resultsSummaryKey:kTookMedicineKey completedTaskPredicate:medicationPredicate reminderBody:NSLocalizedString(@"Take Medication", nil)];
     
     [self.tasksReminder manageTaskReminder:dailySurveyReminder];
     [self.tasksReminder manageTaskReminder:weeklySurveyReminder];
+    [self.tasksReminder manageTaskReminder:dailyFVCSurveyReminder]; //added
```
Comment (ghost) [inline Asthma/Startup/APHAppDelegate.m:214]: No need for comment

**Pull request closed without merging**